wdl/gwas/saige_summary.wdl

```diff
@@ -0,0 +1,116 @@
+task summary{
+    File input_file
+    File finngen_annotation
+    File gnomad_annotation
+    File finngen_tbi = finngen_annotation + ".tbi"
+    File gnomad_tbi = gnomad_annotation + ".tbi"
+    
+    Float pval_thresh
+    String pheno
+    String output_name=pheno+"_summary"
+    String docker
+
+    command <<<
+        python3 <<EOF
+        #read file
+        fname = "${input_file}"
+        finngen_annotation_file = "${finngen_annotation}"
+        gnomad_annotation_file  = "${gnomad_annotation}"
+        sig_threshold = ${pval_thresh}
+        output_name = "${output_name}"
+                    
+        import pysam
+        import gzip
+        from typing import NamedTuple
+
+        FGAnnotation = NamedTuple('FGAnnotation',[('gene',str),('consequence',str)])
+        GnomadAnnotation = NamedTuple('GnomadAnnotation',[('finaf',str),('nfeaf',str),('rsid',str)])
+
+        def get_header(reader,file):
+            with reader(file, "rt") as f:
+                l = f.readline()
+                return l.strip("\n").split("\t")
+
+        def get_fg_annotation(iterator,variant,gene_idx, consequence_idx) -> FGAnnotation:
+            for v in iterator:
+                cols = v.strip("\n").split("\t")
+                if cols[0] == variant:
+                    return FGAnnotation(cols[gene_idx],cols[consequence_idx])
+            return FGAnnotation("","")
+
+        def get_gnomad_annotation(iterator,cpra,finaf_idx, nfeaf_idx, rsid_idx) -> GnomadAnnotation:
+            for v in iterator:
+                cols = v.strip("\n").split("\t")
+
+                if (cols[3] == "chr"+cpra[0]) and (cols[4] == str(cpra[1])) and (cols[5] == cpra[2]) and (cols[6] == cpra[3]):
```

Comment:
no need to necessarily fix now but here again better get the annotation columns from header instead of cols[hardcode]